Tools/CISupport/ews-build/steps.py

```diff
@@ -4123,12 +4137,10 @@ def evaluateCommand(self, cmd):
             self.build.results = SUCCESS
             if RunWebKitTestsInStressMode.FAILURE_MSG_IN_STRESS_MODE not in previous_build_summary:
                 self.setProperty('build_summary', message)
-            self.build.addStepsAfterCurrentStep([ArchiveTestResults(),
-                                                UploadTestResults(),
-                                                ExtractTestResults()])
+            steps_to_add += [ArchiveTestResults(), UploadTestResults(), ExtractTestResults()]
```

Comment:
please verify it this works as self.finished() is called before self.build.addStepsAfterCurrentStep()

Reply:
Yes it works!